ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -2992,6 +2996,8 @@
 				GCC_NO_COMMON_BLOCKS = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				PRODUCT_BUNDLE_IDENTIFIER = "org.reactivecocoa.$(PRODUCT_NAME:rfc1034identifier)-Tests";
+				PRODUCT_NAME = "$(PROJECT_NAME)";
```

Comment:
You need to prefix `Tests` here as well. I checked out your branch and the tests don't build without it.